web/files/referencedby.go

```diff
@@ -10,6 +11,53 @@ import (
 	"github.com/labstack/echo/v4"
 )
 
+// ContentType is the official mime-type for JSON-API
+const ContentType = "application/vnd.api+json"
+
+// UpdatedReferences is a container for the updated list of referenced_by object
+// for an io.cozy.files document.
+type UpdatedReferences struct {
+	DocRev       string                 `json:"rev"`
+	ReferencedBy []couchdb.DocReference `json:"referencedBy"`
+}
+
+func NewUpdatedReferences(rev string, refs []couchdb.DocReference) UpdatedReferences {
```

Comment:
The linter won't be happy with an exported function that has no comment.

Reply:
Right, I forgot about this one (I added it last and didn't run the linter…)